static/src/stylesheets/module/_release-message.scss

```diff
@@ -495,6 +495,166 @@ $btn-height: gs-height(1) - $gs-baseline/2; //30px
 }
 
 /*
+ * Prominent membership messaging AB tests
+ */
+.site-message--membership-message.prominent-level-1 {
+    .site-message__message {
+        padding-left: 10px;
+    }
+    .site-message__media {
+        display: table-cell;
+        vertical-align: top;
+        @include mq($from: mobile) {
+            padding-top: 4px;
+        }
+        @include mq($from: tablet) {
+            padding-top: 1px;
+        }
+    }
+    .site-message__close {
+        vertical-align: top;
+        width: 36px;
+        height: 36px;
+        padding-right: 20px;
+    }
+    .site-message__inner {
+        padding-top: 10px;
+        padding-left: 5px;
+        padding-right: 5px;
+        padding-bottom: 5px;
+    }
+    .site-message__message--membership {
+        margin-right: 5px;
+        font-weight: bold;
```

Comment:
Ben Wuersching is very keen to keep the boldish-ness, and points out that we use "bold" `£` on Membership, where they work:

![image](https://cloud.githubusercontent.com/assets/52038/19596413/692d7fc0-9786-11e6-9668-3cff22234599.png)

...but that's not actually `bold`, it's `font-weight: 900`. If we switch to that the `£` will presumably work. Ben reassures me that Alex Breuer has seen this use of emboldened text, and is fine with it.